wiki/BBCode/en.md

```diff
@@ -187,7 +187,7 @@ The `[profile]` tag is used to link to a user's osu! profile page by using their
 
 *Note: The user ID is a string of numbers that directly follow the `/users/` in the URL of an osu! profile page.*
 
-If specified through their user ID, the text between the open and closed tags will not be parsed and will instead display as the user's current username. However, if the profile page is specified only through a username, and said user changes their username, the link will cease to function.
+If specified through their user ID, the text between the open and closed tags will not be parsed and will instead display as the user's current username. In case there is no text between the open and closed tags specified, the link to the user profile will not be generated correctly. However, if the profile page is specified only through a username, and said user changes their username, the link will cease to function.
```

Comment:
Here for JP>EN translation check. I do not have any knowledge about the topic so please feel free to change it.

When specifying users only by their user IDs, a placeholder text is required between the open and closed tags. Doing so will display the username on the actual page, not the placeholder text. When specifying users only by their username, the link will not function once they change their username.

Reply:
```suggestion
When specifying users only by their user IDs, a placeholder text is required between the open and closed tags. Doing so will display the username on the actual page, not the placeholder text. When specifying users only by their username, the link will not function once they change their username.
```